Traceback (most recent call last):
  File "/www/kirschner.med.harvard.edu/docroot/genomes/code/0j/0j.py", line 51, in <module>
    import kjbuckets
ImportError: libpython2.6.so.1.0: cannot open shared object file: No such file or directory
